cumulus_library_opioid/__init__.py

```diff
@@ -0,0 +1 @@
+__version__="1.0.0"
```

Comment:
Oh - this doesn't have a corresponding change in pyproject (nor any changes in pyproject)